src/image-header.js

```diff
@@ -65,6 +73,17 @@ function parseAppMarkers(dataView) {
         } else if (isApp13PhotoshopMarker(dataView, appMarkerPosition)) {
             fieldLength = dataView.getUint16(appMarkerPosition + APP_MARKER_SIZE, false);
             iptcDataOffset = appMarkerPosition + IPTC_DATA_OFFSET;
+        } else if (isApp2ICCMarker(dataView, appMarkerPosition)) {
+            fieldLength = dataView.getUint16(appMarkerPosition + APP_MARKER_SIZE, false);
+            const iccDataOffset = appMarkerPosition + APP2_ICC_DATA_OFFSET;
+            const iccDataLength = fieldLength - (APP2_ICC_DATA_OFFSET - APP_MARKER_SIZE);
+
+            const iccChunkNumber = dataView.getUint8(appMarkerPosition + ICC_CHUNK_NUMBER_OFFSET, false);
+            const iccChunksTotal = dataView.getUint8(appMarkerPosition + ICC_TOTAL_CHUNKS_OFFSET, false);
+            if (!iccChunks) {
+                iccChunks = [];
+            }
+            iccChunks.push({offset: iccDataOffset, length: iccDataLength, chunkNumber: iccChunkNumber, chunksTotal: iccChunksTotal});
```

Comment:
I would like tests for this block. At least something like `should recognize IPTC data` but probably also something with multiple chunks.

Reply:
added